pkg/cmd/pipelinerun/log_reader.go

```diff
@@ -121,11 +129,35 @@ func (lr *LogReader) readAvailableLogs(pr *v1alpha1.PipelineRun) (<-chan Log, <-
 
 			pipeLogs(logC, errC, tlr)
 		}
+		if pr.Status.Conditions[0].Status == corev1.ConditionFalse {
+			errC <- fmt.Errorf(pr.Status.Conditions[0].Message)
+		}
 	}()
 
 	return logC, errC, nil
 }
 
+func (lr *LogReader) waitUntilAvailable(timeout int, limit int) {
```

Comment:
I would rather use `time.Duration` for those :angel: (and we might to pass a `ctx` too, but that's unrelated to this PR)

Reply:
Thanks for this, used time.Duration.